core/src/main/java/com/novoda/noplayer/player/PlayerFactory.java

```diff
@@ -41,21 +43,25 @@ public Player create() {
     }
 
     public Player create(DrmType drmType, DrmHandler drmHandler) {
+        return create(drmType, drmHandler, USE_SECURE_CODEC);
+    }
+
+    public Player create(DrmType drmType, DrmHandler drmHandler, boolean useSecureCodec) {
         for (PlayerType player : prioritizedPlayerTypes) {
             if (player.supports(drmType)) {
-                return createPlayerForType(player, drmType, drmHandler);
+                return createPlayerForType(player, drmType, drmHandler, useSecureCodec);
             }
         }
         throw UnableToCreatePlayerException.unhandledDrmType(drmType);
     }
 
-    private Player createPlayerForType(PlayerType playerType, DrmType drmType, DrmHandler drmHandler) {
+    private Player createPlayerForType(PlayerType playerType, DrmType drmType, DrmHandler drmHandler, boolean useSecureCodec) {
```

Comment:
https://github.com/novoda/no-player/pull/52/files/4748c1dd02defdb565cd7174a38deb90ee214d9d#r124011428

this would no longer be needed as 99% of cases would never need to manually downgrade the secure decoder 🐱